src/sas/system/config/config.py

```diff
@@ -180,7 +180,15 @@ def __init__(self):
 
         # Time out for updating sasview
         self.UPDATE_TIMEOUT = 2
+        # If True, use an ugly but more robust legend plotting method in Fitting that results in full-
+        # width legends.
+        self.FITTING_PLOT_FULL_WIDTH_LEGENDS = False
```

Comment:
If I'm not mistaken, these changes will affect all plots that aren't in the CorFunc perspective. Having `FITTING` in the name suggests this only pertains to the fitting perspective.

Suggestion: Create a separate configuration file in `sas.system` called `plotting.py` to house plotting-specific configuration values. You will need to then modify `sas.system.__init__.__all__` to include `plotting`.

```suggestion
   class PlottingConfig():
        self.PLOT_FULL_WIDTH_LEGENDS = False
        self.PLOT_LEGEND_TRUNCATE = False
        self.PLOT_LEGEND_MAX_LINE_LENGTH = 30
        
        
    plotting = PlottingConfig()
```